recipes/xz/xz.inc

```diff
@@ -0,0 +1,49 @@
+DESCRIPTION = "XZ Utils is free general-purpose data compression software with a high compression ratio, the successor to LZMA Utils"
+HOMEPAGE = "http://tukaani.org/xz/"
+
+COMPATIBLE_HOST_ARCHS = ".*linux"
+RECIPE_TYPES = "machine native"
+
+# The COPYING file starts
+#
+#   Different licenses apply to different files in this package. Here
+#   is a rough summary of which licenses apply to which parts of this
+#   package (but check the individual files to be sure!):
+#
+# Boiling that down to a single license tag is not easy. Most files
+# are public domain, the remaining are under some GPL variant.
+
+LICENSE = "GPLv2"
```

Comment:
reading in the upstream repo, you at least need to list LGPL-2.1 and GPL-3.0 and "public domain". The latter is really bad actually, since it means there is no proper licensing and you have no clue if it is ok to use the code.

It seems like the license could even be conditional for the different packages, but this is not generally something we have done